hydra/core/override_parser/hpo.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from dataclasses import dataclass
+from typing import List, Union
+
+from hydra.core.override_parser.types import IntervalSweep, Override, Transformer
+
+ValueType = Union[str, int, float]
+
+
+@dataclass
+class HPOParameter:
+    name: str
+    type: str
+    log: bool
+    values: List[ValueType]
+    tags: List[str]
```

Comment:
Not all tag are common across the plugins. For example, NG uses `ordered` tag. We can either make that an attribute of the class or keep that in `tags`